common/src/main/java/com/wynntils/features/user/CustomCharacterSelectionScreenFeature.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.gui.screens.CharacterSelectorScreen;
+import com.wynntils.mc.event.ScreenOpenedEvent;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wynn.event.WorldStateEvent;
+import com.wynntils.wynn.model.WorldStateManager;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class CustomCharacterSelectionScreenFeature extends UserFeature {
+    @Config
+    public boolean onlyOpenOnce = true;
```

Comment:
How does this work? I thought this was the reason you do the openedInThisCharacterSelectionState dance, but you never check this?

Reply:
Oops.